### PR TITLE
chore: add CONTRIBUTING.md

### LINKED ISSUE #431: add CONTRIBUTING.md
We will need a `CONTRIBUTING.md` file. Looks like Projen will not generate one for us. 😄

### PR DESCRIPTION
**Pull request by pahud**
Fixes #431 

<a href="https://gitpod.io/#https://github.com/pahud/cdk-remote-stack/pull/438"></a>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+GitHub provides additional document on [forking a repository](https://help.github.com/articles/fork-a-repo/) and
+[creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
+
+
+## Finding contributions to work on
+Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels (enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any 'help wanted' issues is a great place to start.
+
+
+## Security issue notifications
+If you discover a potential security issue in this project, please reach out to us via the authoer email in [package.json](./package.json). Please do **not** create a public github issue.
```
Comment (alexpulver) [inline CONTRIBUTING.md:48]: Typo: "auhtoer" > "author"

Comment (alexpulver) [reply CONTRIBUTING.md:48]: Capitalization: "github" > "GitHub"

**Comment by pahud** [comment]
Hi @alexpulver please take a look and `approve` this PR if it's good. The approval should trigger the auto merge and auto release.

File: CONTRIBUTING.md (hunk 1 above)
Comment (pahud) [reply CONTRIBUTING.md]: > Capitalization: "github" > "GitHub"

OK on the way.

Comment (pahud) [reply CONTRIBUTING.md]: Both fixed. :-)

**Review by alexpulver — APPROVED** [review]

**Pull request merged**